tritonserver/samples/Simple.java

```diff
@@ -821,10 +818,10 @@ public void reset(Pointer p) {
               cudaMemcpy(dst, input1_data, input1_size, cudaMemcpyHostToHost),
               "setting INPUT1 data in pinned memory");
         }
-      }
 
-      input0_base = use_cuda_memory ? input0_gpu : input0_data;
-      input1_base = use_cuda_memory ? input1_gpu : input1_data;
+        input0_base = use_cuda_memory ? input0_gpu : input0_data;
+        input1_base = use_cuda_memory ? input1_gpu : input1_data;
```

Comment:
We might as well also remove the redundant check for use_cuda_memory here?
BTW, the only reason I did it this way is because the original code is also like that:
https://github.com/triton-inference-server/server/blob/main/src/servers/simple.cc
So that should probably be "fixed" there as well...

Reply:
Nice find! Removed. I'll update in simple.cc as well, thanks for the heads up!